exdb/templates/exdb/home.html

```diff
@@ -0,0 +1,61 @@
+{% extends "exdb/base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="{% static 'exdb/css/home.css' %}"/>
+{% endblock %}
+{% block content %}
+    {{ block.super }}
+    <h3 class="log-out"><a href="{% url 'logout' %}">{% trans "Log out" %}</a></h3>
+    <h2>{% trans 'Hello' %} {{ user }}</h2>
+    <p><a href={% url 'create_experience' %}>{% trans 'Create an experience' %}</a></p>
+    <hr class="break" />
+    <div class="experience-panel">
+        {% for status, experiences in experience_dict.items %}
+            {% if experiences|length %}
+                <div class='experience-div'>
+                    <h1 class="status">{{ status }}</h1>
+                    {% for ex in experiences %}
+                        <div class="experience">
+                            <a href=
+                                "{% if ex.needs_evaluation %}
+                                    {% url 'conclusion' pk=ex.pk %}
+                                {% elif ex.next_approver_id == user.pk and ex.status == 'pe' %}
```

Comment:
You may have to help me with that one. I will not be able to put much time into it for the next couple of weeks.